app/src/main/java/app/revanced/integrations/settings/SettingsEnum.java

```diff
@@ -102,6 +102,7 @@ public enum SettingsEnum {
     HIDE_WATCH_IN_VR("revanced_hide_watch_in_vr", BOOLEAN, FALSE, true),
     PLAYER_POPUP_PANELS("revanced_player_popup_panels_enabled", BOOLEAN, FALSE),
     SPOOF_APP_VERSION("revanced_spoof_app_version", BOOLEAN, FALSE, true, "revanced_spoof_app_version_user_dialog_message"),
+    SPOOF_APP_VERSION_TARGET("revanced_spoof_app_version_target", STRING, "17.30.35", true, parents(SPOOF_APP_VERSION)),
```

Comment:
This would be difficult to keep updated, maybe keep it null to default to the current app version.